src/main/java/org/commcare/formplayer/beans/menus/EntityDetailListResponse.java

```diff
@@ -76,6 +83,20 @@ public EntityDetailListResponse(Detail[] detailList, EvaluationContext ec, TreeR
         return ret;
     }
 
+    private void setSmartLinkParams(EvaluationContext ec) {
+        try {
+            XPathExpression paramExpr = XPathParseTool.parseXPath("commcare_project");
```

Comment:
This is the one place in the formplayer code that references a project/domain. For the sake of being more generic, I'm thinking about pulling this into another app profile property, like a `cc-smart-link-param` property that would accept an XPath expression.

Reply:
Leaving this alone for now.